scripts/testing/city_runner/test_info.py

```diff
@@ -317,6 +358,9 @@ def finish(self, profile):
         not_runs = []
         self.fail_list = []
         self.timeout_list = []
+        self.xfail_unexpectedly_passed_list = []
```

Comment:
There's a common enough term for xfailed tests that end up passing that we could use:
```suggestion
        self.xpass_list = []
```

Reply:
good idea